### PR TITLE
Add support for the VSX and Altivec features on PowerPC

### PR DESCRIPTION
**Pull request by lu-zero**

### CODE REVIEW

File: src/etc/platform-intrinsics/powerpc.rs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (mzji) [inline src/etc/platform-intrinsics/powerpc.rs]: I think we should use ".json" as the extension of this file.

**Comment by lu-zero** [comment]
Do you prefer having the intrinsics being added one by one or in a single commit ?

File: src/etc/platform-intrinsics/powerpc.rs (hunk 1 above)
Comment (lu-zero) [inline src/etc/platform-intrinsics/powerpc.rs]: Yes, I did not consider the fact there was a pull request open on the master branch when I pushed it...

**Comment by alexcrichton** [comment]
@bors: r+

Nah seems fine!

**Comment by Mark-Simulacrum** [comment]
@bors retry 

Synchronized bors, and it forgot about this PR being tested...

**Comment by lu-zero** [comment]
The failure seems unrelated to anything I touched with those two patches...

**Comment by Mark-Simulacrum** [comment]
@bors retry
  - #43453

```
[01:21:51] 	thread 'test::test_bin_lib_project_no_cfg_test' panicked at 'Hit timeout', /checkout/src/tools/rls/src/test/harness.rs:142:12
```

**Pull request merged**